networking/enable-cluster-wide-proxy.adoc

```diff
@@ -0,0 +1,17 @@
+[id="enable-cluster-wide-proxy"]
+= Configuring the cluster-wide proxy
+include::modules/common-attributes.adoc[]
+:context: config-cluster-wide-proxy
+
+toc::[]
+
+Production environments can deny direct access to the Internet and instead have
+an HTTP or HTTPS proxy available. You can configure {product-title} to use a
+proxy by
+xref:../networking/enable-cluster-wide-proxy.adoc#nw-proxy-configure-object_config-cluster-wide-proxy[modifying the Proxy object for existing clusters]
+or by configuring the proxy settings in the `install-config.yaml` file for new
```

Comment:
hi @bergerhoffer do we have related doc PR for this?  Guiding customer how to configure proxy settings in the `install-config.yaml` is also very important.